src/fr/starboundscans/build.gradle

```
@@ -1,9 +1,9 @@
 ext {
     extName = 'Starbound Scans'
     extClass = '.StarboundScans'
-    themePkg = 'keyoapp'
+    themePkg = 'madara'
     baseUrl = 'https://starboundscans.com'
-    overrideVersionCode = 0
+    overrideVersionCode = 1
```

```suggestion
    overrideVersionCode = 0
```

`overrideVersionCode` can stay zero because Madara's base version code is way higher than the previous Keyoapp.